src/cli/domain/init-template/index.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const path = require('path');
+const scaffold = require('./scaffold');
+const installTemplate = require('./install-template');
+const createComponentDir = require('./create-component-dir');
+const initPackage = require('./init-package');
+const utils = require('./utils');
+
+module.exports = function(options, callback) {
+  const {
+    componentName,
+    templateType,
+    logger = console,
+    cli = 'npm'
+  } = options;
+  const componentPath = path.join(process.cwd(), componentName);
+  const local = /^\.+\/|^\//.test(templateType);
+  const template = utils.getPackageName(templateType).replace('-compiler', '');
```

Comment:
why are we doing this? If we want to normalise the name, I would rather add this or the next (compiler) into a sanitise method or something like that, in order to make the reader understand a bit more why we are doing this.